bin/key_server.js

```diff
@@ -74,11 +75,21 @@ function run(config) {
     log.stat(Password.stat())
   }
 
+  P.promisifyAll(redis.RedisClient.prototype)
+  P.promisifyAll(redis.Multi.prototype)
+
+  const redisClient = redis.createClient({
+    host: config.redis.host,
+    port: config.redis.port,
+    prefix: config.redis.prefix
+  })
+
   var DB = require('../lib/db')(
     config,
     log,
     Token,
-    UnblockCode
+    UnblockCode,
+    redisClient
```

Comment:
@udaraweerasinghege I think we want to move away from `functional approach` here, AKA requiring stuff in the server.js and then passing it through. Can we just require the redisClient in the files we need it?